cmd/samples/http/sender-with-custom-client/main.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+)
+
+const (
+	count = 1000
+)
+
+type envConfig struct {
+	// Target URL where to send cloudevents
+	Target string `envconfig:"TARGET" default:"https://localhost:8080" required:"true"`
```

Comment:
like here: this allows you to validate that an env var is correct and set, so you can mark yours as required for this to work.

Re: "CLIENT_KEY" and "CLIENT_CERT"

Reply:
absolutely